cristin-organization/src/main/java/no/unit/nva/cristin/organization/CristinOrganizationApiClient.java

```diff
@@ -80,6 +90,41 @@ public Organization getOrganization(URI uri)
         return fromSubSubunit(getSubSubUnitDtoWithMultipleEfforts(uri));
     }
 
+    /**
+     * Get information for an Organization without parent of subunits.
+     *
+     * @param identifier the Cristin unit URI
+     * @return an {@link Organization} containing the information
+     * @throws NotFoundException when the URI does not correspond to an existing unit.
+     */
+    public Organization getFlatOrganization(String identifier) throws ApiGatewayException, InterruptedException {
+        URI cristinUri = new UriWrapper(CRISTIN_API_URL)
+                .addChild(UNITS_PATH)
+                .addQueryParameter(UNIT_ID, identifier)
+                .getUri();
+
+        HttpResponse<String> response = sendRequestMultipleTimes(addLanguage(cristinUri)).get();
+        if (isSuccessful(response.statusCode())) {
+            try {
```

Comment:
Do all these lines need to be in the try catch? Or can you use attempt() orElseThrow?